config/initializers/assets.rb

```diff
@@ -6,8 +6,14 @@
 # Precompile additional assets.
 # application.js, application.css, and all non-JS/CSS in app/assets folder are already added.
 # Rails.application.config.assets.precompile += %w( search.js )
-Rails.application.config.assets.precompile += %w( shadowbox.css )
-Rails.application.config.assets.precompile += %w( SortedList.js )
```

Comment:
Does the error message say 'add missing line to asset.rb and restart the server'?